scripts/jenkins/Jenkinsfile.release

```
@@ -677,6 +686,26 @@ def buildRelease(stageName, image, packageRevision='1',
   }]
 }
 
+
+/**
+ * Build OpenWrt packages with the current current git master state.
+ *
+ * @param image which docker image should be used
+ */
+def buildOpenwrtPackage(image) {
+  def stageName = 'build-openwrt-package'
+  return [(stageName): {
+    stage(stageName) {
+      withDockerEnv(image, [DockerOpts.MOUNT_MIRROR]) {
+        checkout scm
+        sh "./scripts/release/prepare-openwrt-makefile.sh /home/build/openwrt/package/feeds/packages/elektra/Makefile"
+        sh "cd /home/build/openwrt && make package/feeds/packages/elektra/compile V=sc"
```

Does this maybe create too many logs? What about first try as fast as possible without verbosity, if it fails without concurrency and full verbosity, e.g.:

```suggestion
        sh "cd /home/build/openwrt && make package/feeds/packages/elektra/compile || make -j1 V=sc package/feeds/packages/elektra/compile"
```

see also #3904